kpet/__init__.py

```diff
@@ -98,6 +98,18 @@ def build_run_command(cmds_parser, common_parser):
         default=[],
         help='List of mbox URLs/paths comprising the patch series'
     )
+    print_test_cases_parser = action_subparser.add_parser(
+        "print_test_cases",
+        help="Print test cases according to the patches",
+        parents=[common_parser],
+    )
+    print_test_cases_parser.add_argument(
+        '-m',
+        '--mboxes',
+        required=True,
+        nargs='*',
+        help='List of mbox URLs/paths comprising the patch series'
```

Comment:
Could you make those positional parameters, instead of a multi-value option?

Also, invoking `kpet run print_test_cases`, as well as `kpet run generate`, without patches should match *all* test cases (for the specified tree/arch/etc., eventually).

Reply:
Done